signalfx-java/pom.xml

```diff
@@ -43,15 +43,37 @@
   </developers>
 
   <dependencies>
-    <!-- compile -->
+    <!-- pure deps for shading -->
+    <dependency>
+      <groupId>com.signalfx.public</groupId>
+      <artifactId>signalfx-connection</artifactId>
+      <version>${version}</version>
+    </dependency>
     <dependency>
-      <groupId>org.eclipse.jetty.websocket</groupId>
-      <artifactId>websocket-client</artifactId>
+      <groupId>com.signalfx.public</groupId>
+      <artifactId>signalfx-endpoint</artifactId>
+      <version>${version}</version>
```

Comment:
These two also appear in the `<!-- compile -->` section; in an identical form.

Reply:
Thanks!